code/game/atoms.dm

```diff
@@ -48,6 +48,8 @@ var/global/list/ghdel_profiling = list()
 	//var/harm_label_icon_state //Makes sense to have this, but I can't sprite. May be added later.
 	var/list/last_beamchecks // timings for beam checks.
 
+	var/vending_cat = null// subcategory for vending machines.

```

Comment:
eh, you're right, not sure why https://github.com/d3athrow/vgstation13/pull/4935/files#diff-be8fc4ef536d83a7c390019dd59d14efR228 was a var/atom when it could have been a var/obj/item. I'm changing that.